.github/workflows/cleanup_cache.yml

```diff
@@ -0,0 +1,63 @@
+name: Cleanup cache
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+
+    permissions:
+      actions: write
+      contents: read
+
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      REPO: ${{ github.repository }}
+      BRANCH: refs/pull/${{ github.event.pull_request.number }}/merge
```

Comment:
I suppose that `${{ github.ref }}` would be the right one?
![image](https://github.com/openvinotoolkit/openvino/assets/65596953/13fa7aba-e8c5-423e-a7eb-b68ce2d471b4)